### PR TITLE
New package: 3u.3utools version 3.03

### PR DESCRIPTION
**Pull request by upintheairsheep**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/118743)

### CODE REVIEW

**Comment by upintheairsheep** [comment]
Interactive Only according to previous attempts to add 3utools to winget

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> 12/7/2023 6:58:34 PM
Installer Verification Analysis Context Information: 
Specified hash doesn't match.
Actual hash:   8FAF1527DCE8FAAD70226659157F81374DBC69D4CB13BEE4B06B6082AAE23440
Expected hash: 964003024551ae014f63cc24c093e43c511f523add59cd2c87764645ee1a2eaf

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> 12/7/2023 6:58:34 PM
Installer Verification Analysis Context Information: 
Specified hash doesn't match.
Actual hash:   8FAF1527DCE8FAAD70226659157F81374DBC69D4CB13BEE4B06B6082AAE23440
Expected hash: 964003024551ae014f63cc24c093e43c511f523add59cd2c87764645ee1a2eaf

File: manifests/3/3u/3utools/3.03/3u.3utools.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: 3u.3utools
+PackageVersion: 3.03
+Installers:
+- InstallerUrl: https://url.3u.com/zmAJjyaa
+  Architecture: x86
+  InstallerType: exe
+  InstallerSha256: 964003024551AE014F63CC24C093E43C511F523ADD59CD2C87764645EE1A2EAF
```
Comment (stephengillie) [inline manifests/3/3u/3utools/3.03/3u.3utools.installer.yaml:10]:
```suggestion
  InstallerSha256: 8FAF1527DCE8FAAD70226659157F81374DBC69D4CB13BEE4B06B6082AAE23440
```

**Pull request closed without merging**